### ISSUE TITLE
Facets ... modal close button missing accessibility label for screen readers

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#3045]
### Descriptive summary

From the Accessibility 360 Audit Report:

> Close (x) button
> This close button is not labeled for screen reader users.

![close-button-missing-sr-tag](https://user-images.githubusercontent.com/3020266/39719533-2376e6bc-51ff-11e8-8407-9817e4a8bac6.png)

### Rationale

Accessibility improvement for screen readers.

### Tasks
- [ ] Label the close button for screen reader users.

### Steps to reproduce the behavior

1. Go to Nurax homepage (https://nurax.curationexperts.com/)
2. Search for anything.
3. When you get a search results display resembling screen shot above, click on "Keyword" facet panel header element.
4. Click the "more >>" link.
5. The (x) close button in upper right-hand corner of modal window is the element referenced in this ticket.

**Comment by blancoj** [comment]
This fix takes place in the Blacklight gem.  Updating the gem would fix this, since the change seems to already be in the master branch:

https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/facet.html.erb#L7

**Comment by kdid** [comment]
This needs a backport to 6.x

**Comment by rjkati** [comment]
WAVE insists on opening the modal in a new window, so it's difficult to use WAVE to test this ticket. 

However, inspecting the HTML shows that the close button currently contains `aria-label="Close"`, so it seems like this is fixed. @jlhardes 
![close_button_model](https://user-images.githubusercontent.com/32644254/108761842-5d9cf180-751d-11eb-9047-2a194b617007.PNG)

**Comment by jlhardes** [comment]
With the aria-label in place, this issue is fixed and closed.
